packages/components/src/components/modal/modal.scss

```diff
@@ -2,7 +2,9 @@
 .dc-modal {
     @include mobile {
         position: absolute;
-        top: 0;
+        top: 50vh;
```

Comment:
Shouldn't this only be applied for when we use the `small` `prop` with `Modal`? Since this positioning only applies to small dialog (Modal with `small`)

Because `AccountSwitcherMobile` and `TradeParamsMobile` are using `Modal` as well with the existing `top: 0`.